system/jlib/jutil.hpp

```diff
@@ -231,9 +231,9 @@ class jlib_decl StringArray : public ArrayOf<const char *, const char *, StringP
     typedef ArrayOf<const char *, const char *, StringPointerArrayMapper> PARENT;
 public:
     // Appends a list in a string delimited by 'delim'
-    void appendList(const char *list, const char *delim, bool trimSpaces = true);
+    void appendList(const char *list, const char *delim, bool trimSpaces = true, unsigned maxNumItems = 0);//maxNumItems = 0: no limit
     // Appends a list in a string delimited by 'delim' without duplicates
-    void appendListUniq(const char *list, const char *delim, bool trimSpaces = true);
+    void appendListUniq(const char *list, const char *delim, bool trimSpaces = true, unsigned maxNumItems = 0);
```

Comment:
not convinced about this change (and changed to DelimToStringArray).

See other comments about parsing out the processID more efficiently. In which case use of this will not be necessary.